Formula/icarus-verilog.rb

```diff
@@ -1,10 +1,10 @@
 class IcarusVerilog < Formula
   desc "Verilog simulation and synthesis tool"
   homepage "http://iverilog.icarus.com/"
-  url "https://github.com/steveicarus/iverilog/archive/v10_3.tar.gz"
-  mirror "https://deb.debian.org/debian/pool/main/i/iverilog/iverilog_10.3.orig.tar.gz"
-  sha256 "4b884261645a73b37467242d6ae69264fdde2e7c4c15b245d902531efaaeb234"
-  license "LGPL-2.1"
+  url "https://github.com/steveicarus/iverilog/archive/v11_0.tar.gz"
+  mirror "https://deb.debian.org/debian/pool/main/i/iverilog/iverilog_11.0.orig.tar.gz"
+  sha256 "6327fb900e66b46803d928b7ca439409a0dc32731d82143b20387be0833f1c95"
+  license "GPL-2.0-or-later"
```

Comment:
This is not correct. There is a `COPYING.lesser` file to indicate it's LGPL

Reply:
But is it `LGPL-2.1-only` or `LGPL-2.1-or-later`?

Reply:
It'd say `-or-later` based on the definitions in the file headers and can you open an issue upstream to check? Because those headers indicate GPL, in conflict with the COPYING files.